src/subject.hpp

```diff
@@ -32,6 +32,7 @@ class Subject {
   void set_color_depth(int depth);
   void set_timezone(const string & timezone);
   void set_language(const string & language);
+  void set_useragent(const string & user_agent);
```

Comment:
Funny to see that different languages have different styles to do the same thing.
Pointer operators in ObjC are suggested near the var name (e.g.: NSString *var) meanwhile in C++ seems preferred near the type (e.g.: string* p). Here we placed it in the middle. Maybe it was the trend back in the 2016 :)